server/connector/connector.go

```diff
@@ -194,22 +272,45 @@ func (c *connector) Post(w http.ResponseWriter, req *http.Request) {
 		} else {
 			http.Error(w, fmt.Sprintf(`{"error":"unknown error: %s"}`, err.Error()), http.StatusInternalServerError)
 		}
+
 		return
 	}
 	go c.Run(subscriber)
 	c.logger.WithField("topic", topic).Info("Subscription created")
 	fmt.Fprintf(w, `{"subscribed":"/%v"}`, topic)
+
+	if errFill == nil {
+		err = event.report(c.KafkaProducer, c.KafkaReportingTopic)
+		if err != nil {
+			logger.WithError(err).Error("Could not report sent subscribe sms to Kafka topic")
+		}
+	}
+
 }
 
 // Delete removes a subscriber
 func (c *connector) Delete(w http.ResponseWriter, req *http.Request) {
+
+	event := SubscribeUnsubscribeEvent{
+		Type: "marketing_notification_subscription_information",
+		Time: time.Now().UTC().Format(time.RFC3339),
+		Payload: SubscribeUnsubscribePayload{
+			Service: c.config.Name,
+			Action:  "unsubscribe",
+		},
+	}
+
 	params := mux.Vars(req)
 	c.logger.WithField("params", params).Info("DELETE subscription")
 	topic, ok := params[TopicParam]
 	if !ok {
 		fmt.Fprintf(w, "Missing topic parameter.")
 		return
 	}
+
+	event.Payload.Topic = topic
+	errFill := event.fillParams(params)
```

Comment:
not really...I would want to fill the event even if it will not be put in KAFKA.